OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -410,11 +415,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <condition key="deficient">
                                         The textblock has been resewn together and parchment guards have been placed around all quires.
                                         <locus from="51r" to="62v"/> have been stitched together at a later stage.
-                                        The textblock has been damaged by humidity, especially in the outer margins, and water stains (e.g. on <locus target="#37 #104v"/>).
+                                        The textblock has been damaged by humidity, especially in the outer margins, and water stains are present (e.g. on <locus target="#37 #104v"/>).
```

Comment:
Above, bookblock is used. Is this depending on whether or not the manuscript contains endleaves?

Reply:
There is such difference but I did not use it consciously. I will switch to bookblock, it is more appropriate I think.